pkg/scheduler/framework/v1alpha1/interface.go

```diff
@@ -184,13 +184,13 @@ type PreFilterPlugin interface {
 	// PreFilter is called at the beginning of the scheduling cycle. All PreFilter
 	// plugins must return success or the pod will be rejected.
 	PreFilter(pc *PluginContext, p *v1.Pod) *Status
-	// Updater returns an updater if the plugin implements one, or nil if it
-	// does not. A Pre-filter plugin can provide an updater to incrementally
-	// modify its pre-processed info. The framework guarantees that the updater
+	// Extensions returns an extension if the plugin implements one, or nil if it
+	// does not. A Pre-filter plugin can provide an extension to incrementally
+	// modify its pre-processed info. The framework guarantees that the extension
 	// AddPod/RemovePod functions will only be called after PreFilter,
 	// possibly on a cloned PluginContext, and may call those functions more than
 	// once before calling Filter again on a specific node.
-	Updater() Updater
+	Extensions() PreFilterExtensions
```

Comment:
I would rephrase the comment like this:

Extensions returns an extensions interface if the plugin implements one, or nil if it does not. A Pre-filter plugin can provide extensions to incrementally modify its pre-processed info. The framework guarantees that the extensions AddPod/RemovePod will only be called after PreFilter,	possibly on a cloned PluginContext, and may call those functions more than once before calling Filter again on a specific node.